tools/profileBuilder/model/transforms.go

```diff
@@ -119,6 +118,9 @@ func BuildProfile(packageList ListDefinition, name, outputLocation string, outpu
 					var ok bool
 					if aliasPath, ok = packageList.PathOverride[importPath]; !ok {
 						var err error
+						if modules && modinfo.HasVersionSuffix(path) {
+							path = filepath.Dir(path)
+						}
```

Comment:
hmm not sure why this is necessary?

Reply:
i guess we dont want the version number as the name

---
In reply to: [290957240](https://github.com/Azure/azure-sdk-for-go/pull/4959#discussion_r290957240) [](ancestors = 290957240)

Reply:
can we add a comment to it ?

---
In reply to: [290957336](https://github.com/Azure/azure-sdk-for-go/pull/4959#discussion_r290957336) [](ancestors = 290957336,290957240)